packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -59,7 +60,7 @@ export const Modal = ({ children, isOpen, onClose }: ModalProps) => {
       onClose={onClose}
       PaperComponent={Paper}
       fullScreen={fullScreen}
-      disablePortal
+      disablePortal={disablePortal}
```

Comment:
Is there a particular reason we would like to set this as false, ever? The main reason we keep it as try is so it stays within the relevant components and we can control styles.